pkgs/top-level/all-packages.nix

```diff
@@ -1345,6 +1345,15 @@ in
 
   duperemove = callPackage ../tools/filesystems/duperemove { };
 
+  dvc = callPackage ../applications/version-management/dvc { };
```

Comment:
Why not use `python3Packages.callPackage`?

Reply:
Nope, we don't want to do this because we might have to override a Python package some day.

Reply:
What do you mean?

Reply:
I mean you shouldn't use `pythonPackages.callPackage` in `all-packages.nix`.